libraries/AP_WindVane/AP_WindVane.h

```diff
@@ -147,15 +133,11 @@ class AP_WindVane
         WINDSPEED_SITL               = 10
     };
 
-    // pin for reading analog voltage
-    AP_HAL::AnalogSource *dir_analog_source;
-    AP_HAL::AnalogSource *speed_analog_source;
-    AP_HAL::AnalogSource *speed_temp_analog_source;
+    // store sensor types we are using so we can change at runtime
+    int8_t _init_direction_type;
```

Comment:
this can probably go if we require a reboot to change drivers

Reply:
fixed